bookwyrm/views/books/books.py

```diff
@@ -15,22 +15,26 @@
 from bookwyrm.connectors import connector_manager
 from bookwyrm.connectors.abstract_connector import get_image
 from bookwyrm.settings import PAGE_LENGTH
-from bookwyrm.views.helpers import is_api_request
+from bookwyrm.views.helpers import is_api_request, maybe_redirect_local_path
 
 
 # pylint: disable=no-self-use
 class Book(View):
     """a book! this is the stuff"""
 
-    def get(self, request, book_id, user_statuses=False):
+    def get(self, request, book_id, **kwargs):
```

Comment:
what the advantage of using kwargs here? is that because of the changes to urls.py?

Reply:
So I believe it's only relevant for API requests at the moment (and came up in unit tests of `views/books/test_book.py`) but before this change the view supported `view(request, self.book.id, "review")` (or `comment`/`quotation`) to filter user statuses -- but since `user_statuses` was an optional argument and `slug` is also an optional argument, there's no way to differentiate between those args without specifying  both. So I'm using kwargs instead so that they're named.

An alternative would be to force an ordering. If we did that, then if we specified user_statuses first we'd have to pass `False` around everywhere we wanted to specify the slug, and (afaik) it would need to be in the URL. So that doesn't seem right.

Reply:
I see! Thanks for explaining -- I think the way you have it works :)

Reply:
This is used outside the API--for the tabs at the bottom of the book page.